dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -449,13 +449,11 @@ DMContextPtr DeltaMergeStore::newDMContext(
     const String & tracing_id,
     ScanContextPtr scan_context_)
 {
-    std::shared_lock lock(read_write_mutex);
-
```

Comment:
The lock was added to protect the "schema" in very old versions(https://github.com/pingcap/tiflash/pull/507/files#diff-22b900e9e8020dc835612316f3bf151cae28b621bfac64d6a22b377d062f4b7eR165-R190). But now the members used in this function do not 
have concurrent issues. So this lock can be removed.